source/plugin/blocks/modifying.rst

```diff
@@ -96,17 +101,18 @@ The following example will dry the block at a given ``Location``, if possible.
         }
     }
 
-Since the ``WetData`` data manipulator represents boolean data, by removing it we set the wetness of the block (if it
-has any) to false. The ``dryState.isPresent()`` check will fail on block states that can not be wet since ``dryState``
-will be ``Optional.empty()`` in that case.
+Since the :javadoc:`org.spongepowered.api.data.manipulator.mutable.WetData` data manipulator represents boolean data,
+by removing it we set the wetness of the block (if it has any) to false. The ``dryState.isPresent()`` check will fail
+on block states that can not be wet since ``dryState`` will be ``Optional.empty()`` in that case.
 
 Copying Blocks
 ~~~~~~~~~~~~~~
 
-If you want to copy all of a blocks data, the ``BlockSnapshot`` class is your best friend. While it doesn't expose all
-the data, it stores a ``BlockType``, its ``BlockState`` and, if necessary, all additional Tile Entity Data (for example chest
-inventories). Conveniently, the ``Location`` class provides a ``createSnapshot()`` method to create a snapshot of the block
-at that point in time. That makes copying blocks from one location to another very simple:
+If you want to copy all of a blocks data, the :javadoc:`org.spongepowered.api.block.BlockSnapshot` class is your best
```

Comment:
That should be "a block's data" - please use the [possessive apostrophe](http://www.thepunctuationguide.com/apostrophe.html) where appropriate.